resources/user-agents/browsers/firefox/firefox-0-x.json

```diff
@@ -81,79 +83,6 @@
           "platforms": [
             "SunOS"
           ]
-        },
```

Comment:
As mentioned above, I removed these rules and put them in their own Firebird file.